maketestconfig.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+# Usage:
+#
+#      ./makeflash.sh <output.img>
+#
+MKCONFIG_TAG="$(linuxkit pkg show-tag pkg/test-cert)-amd64"
+
+# Make sure it exists
```

Comment:
I don't see a check for existence

Reply:
Yes, artifact from previous iterations. Creating it here showed problems on Mac.